### PR TITLE
skale implementation on fortune app

### PR DESCRIPTION
**Pull request by AzizBardi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -40,8 +34,8 @@ services:
       context: ./recording-oracle
     network_mode: "host"
     environment:
-      ETH_PRIVATE_KEY: 486a0621e595dd7fcbe5608cbbeec8f5a8b5cabe7637f11eccfc7acd408c3a0e
-      ETH_HTTP_SERVER: http://localhost:8547
+      ETH_PRIVATE_KEY: 09762fa0f91726089540420b0a83c01048eb6b2ba178cafea7c12e5d96c57896
+      ETH_HTTP_SERVER: https://staging-v2.skalenodes.com/v1/stocky-pleione
```
Comment (posix4e) [inline docker-compose.yml:38]: Why is this needed?

**Review by posix4e — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (AzizBardi) [reply docker-compose.yml:38]: These are the private key and the Skale testnet config.
